### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by 7thRA-ONE** [WhiskeySockets/Baileys#842]
Problem:
Messages cannot be seen in some groups, while the bot logs that the message is sent successfully. However, the bot works fine in other groups and private chats.

Environment:
- `@whiskeysockets/baileys`: ^6.7.1
- Node.js: ^20.12.2
- npm: ^10.5.2
- Platform: Heroku .

Any solution???

**Comment by spacesoft321** [comment]
yes group message api and group message autoreply not good working
